knowledge-base/how-to-use-kendodatetimepicker-for-parameter-editors-in-the-angular-report-viewer.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Use KendoDateTimePicker for Parameter Editors in the Angular Report Viewer
+description: See how to use the KendoDateTimePicker for selecting date and time for the report parameter by creating custom parameter editor in the Angular Report Viewer
+type: how-to
+page_title: Using KendoDateTimePicker in Angular Report Viewer
+slug: how-to-use-kendodatetimepicker-for-parameter-editors-in-the-angular-report-viewer
+position: 
+tags: angular, reportviewer, kendodatetimepicker
+ticketid: 1543447
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Project Type</td>
+			<td>Angular</td>
+		</tr>
+		<tr>
+			<td>Report Viewer</td>
+			<td>Angular Report Viewer</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+This article describes how to use the [KendoDateTimePicker](https://docs.telerik.com/kendo-ui/api/javascript/ui/datetimepicker) as a custom parameter editor
+in [Angular Report Viewer](./angular-report-viewer). This approach allows the selection of date and time for the **DateTime** [report parameters](./designing-reports-parameters).
+
+
+## Solution
+Telerik kendoDateTimePicker widget is not included in the Kendo UI JavaScript distributed by 
+Telerik Reporting through telerikReportViewer.kendo-x.x.x.min.js file, or kendo.subset.2015.3.930.min.js in older versions.
+For that reason, you will have to replace the Kendo UI subset with the full Kendo UI JavaScript e.g. kendo.all.min.js.
+
+Also, you need to make sure that Kendo all is loaded after jQuery. Because of that, we will use a special logic that sets the visibility
```

Comment:
You may put this as a note so that it is easily noticeable since it feels like a very important requirement.
You can do this by putting > at the beggining